Different TCP/IP addresses and protocols

openBackhaul opened the pull request:
Add tcp-server-interface-configuration::description
Add tcp-server-interface-configuration::protocol
Update OAS and CONFIGfile
Expand FakeIpList by 2nd per application

Fixes: #331

IswaryaaS reviewed (changes requested):
The updated file looks fine.
Additionally, the [ApplicationPattern.services.yaml](https://github.com/openBackhaul/ApplicationPattern/blob/develop/ApplicationPattern%2Bservices.yaml) need to be complemented with additional tcp-server instance information. 
Also, in docs, [ElementsOfApplicationPattern/LogicalTerminationPoint](https://github.com/openBackhaul/ApplicationPattern/blob/develop/doc/ElementsApplicationPattern/InformationModel/LogicalTerminationPoint/LogicalTerminationPoint.md) file would need an update about the additional tcp-server instance.

PrathibaJee commented on ApplicationPattern.yaml:
```diff
@@ -8605,6 +8635,160 @@ paths:
+        required: true
+        schema:
+          type: string
+          pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'
```
The current pattern , will accept only "ro-2-0-0-tcp-s-bm-000". Since we have another tcp-server instance now , we require a pattern that accepts "ro-2-0-0-tcp-s-bm-001" too. Kindly please consider the following proposal, 
**Proposal :** 
Requesting to change the Pattern from , 
`pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'`
to 
`pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-00[0-1]`

openBackhaul replied:
CTRL + H and changed as prosed. Thanks

PrathibaJee commented on ApplicationPattern.yaml:
```diff
@@ -8605,6 +8635,160 @@ paths:
+                  type: string
+      responses:
+        '204':
+          description: 'Local port documented'
```
Since this Put request documents the description , can we change the description from 
'Local port documented'
to 
'description documented' .

openBackhaul replied:
Changed as proposed.

PrathibaJee commented on ApplicationPattern.yaml:
```diff
@@ -8605,6 +8635,160 @@ paths:
+        required: true
+        schema:
+          type: string
+          pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'
```
Same proposal for the pattern as similar to the one provided for TcpServerDescription.
**Proposal :** 
Requesting to change the Pattern from , 
`pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-000'`
to 
`pattern: '^([a-z]{2,6})-([0-9]{1,2})-([0-9]{1,2})-([0-9]{1,2})-tcp-s-bm-00[0-1]`

PrathibaJee replied:
Also kindly include the same pattern to all the OAM services related to the tcp-server path parameters. Thanks.

openBackhaul replied:
Changed as proposed.

PrathibaJee commented on ApplicationPattern.yaml:
```diff
@@ -8605,6 +8635,160 @@ paths:
+                    - 'tcp-server-interface-1-0:PROTOCOL_TYPE_NOT_YET_DEFINED'
+      responses:
+        '204':
+          description: 'Local port documented'
```
Since this Put request documents the description , can we change the description from
'Local port documented'
to
'protocol documented' .

openBackhaul replied:
Changed as proposed.

PrathibaJee commented on doc/TestingApplications/Infrastructure/SdnLaboratory/FakeAddresses/IpAddresses.md:
```diff
@@ -2,9 +2,9 @@
 
```
Many thanks for covering the changes in this file.

openBackhaul replied:
Excited to see whether this FakeIp thing will fly.

PrathibaJee reviewed (changes requested):
Provided few review comments. Kindly please resolve incase if it is valid. Many thanks.

openBackhaul commented:
> [ApplicationPattern.services.yaml](https://github.com/openBackhaul/ApplicationPattern/blob/develop/ApplicationPattern%2Bservices.yaml) need to be complemented with additional tcp-server instance information.  

Changed as proposed.

> [ElementsOfApplicationPattern/LogicalTerminationPoint](https://github.com/openBackhaul/ApplicationPattern/blob/develop/doc/ElementsApplicationPattern/InformationModel/LogicalTerminationPoint/LogicalTerminationPoint.md) file would need an update about the additional tcp-server instance.

Changed as proposed.

PrathibaJee reviewed (approved):
Thanks for resolving the review comments. Approving the changes as there is no further comments.
